docs_src/microservices/device/supported/device-usb-camera/walkthrough/deployment.md

```diff
@@ -109,6 +109,74 @@ Follow this guide to deploy and run the service.
     !!! Note 
         The `jq -r` option is used to reduce the size of the displayed response. The entire device with all information can be seen by removing `-r '"deviceName: " + '.devices[].name'', and replacing it with '.'`
 
+1. Add credentials to for the rtsp stream.
+
+    !!! note
+        If you want to disable rtsp authentication entirely, you must [build a custom image](../walkthrough/custom-build.md).
+
+    === "Scripts"
+
+        !!! note 
+            If running in secure mode the [JWT token](../../../../../security/Ch-APIGateway.md#creating-access-token-for-api-gateway-authentication) generated previously is needed for mapping credentials.
+
+        a. Enter JWT token if in secure mode.  
+            ![](../images/enter-jwt-token.png)  
+        b. Set the username.  
+            ![](../images/set-username.png)  
+        c. Set the password.  
+            ![](../images/set-username.png)  
+
+        Successful:
+        ```bash
+        Dependencies Check: Success
+            Consul Check: ...
+                            [
+        {
+            "Resource": "key",
+            "Access": "read",
+            "Allow": false
+        },
+        {
+            "Resource": "key",
+            "Access": "write",
+            "Allow": false
+        }
+        ]
+
+        curl --data '{
+            "apiVersion":"v3",
+            "secretName": "rtspauth",
+            "secretData":[
+                {
+                    "key":"username",
+                    "value":"username"
+                },
+                {
+                    "key":"<redacted>",
```

Comment:
change to `password`. the reason it was redacted was because your password was password ;)
```suggestion
                    "key":"password",
```